3party/opening_hours/opening_hours.cpp

```diff
@@ -883,6 +883,13 @@ OpeningHours::OpeningHours(TRuleSequences const & rule):
 {
 }
 
+OpeningHours::OpeningHours(TRuleSequences const & rule, std::unordered_map<std::string, std::string> const & holidays):
```

Comment:
Do not use `std::unordered_map<std::string, std::string>`. Create a typename like `TRuleSequences`. When you decide to change unordered_map to something different you don't want to go through all files where it is used

Reply:
std::vector<time_t> here should be more than enough